### PR TITLE
888-UseCentralServicesSharedLib

### PR DESCRIPTION
**Pull request by gibaros**
-use shared lib enums
-use latest central-services-shared version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@hapi/good": "8.2.1",
     "@hapi/hapi": "18.3.2",
+    "@mojaloop/central-services-shared": "7.4.3",
```
Comment (rmothilal) [inline package.json]: please update all dependencies and version of the package.json

File: src/model/quotes.js [hunk 2]
```diff
@@ -740,15 +744,15 @@ class QuotesModel {
...
-          'fspiop-destination': fspiopSource,
-          'fspiop-source': 'switch',
-          'fspiop-http-method': 'PUT'
+        headers: (headers || {
+          'Enum.Http.Headers.FSPIOP.DESTINATION': fspiopSource,
+          'Enum.Http.Headers.FSPIOP.SOURCE': 'Enum.Http.Headers.FSPIOP.SWITCH.value',
+          'Enum.Http.Headers.FSPIOP.HTTP_METHOD': 'Enum.Http.RestMethods.PUT'
         }, true)
       }
       let res
```
Comment (rmothilal) [inline src/model/quotes.js]: Please use the shared library sendRequest function else you have to keep the generateRequestHeaders function

File: src/model/quotes.js [hunk 3]
```diff
@@ -487,9 +489,9 @@ class QuotesModel {
...
 
       let opts = {
-        method: 'PUT',
+        method: Enum.Http.RestMethods.PUT,
         body: JSON.stringify(originalQuoteResponse),
-        headers: this.generateRequestHeaders(headers)
+        headers: headers
       }
 
       // Network errors lob an exception. Bare in mind 3xx 4xx and 5xx are not network errors
```
Comment (rmothilal) [inline src/model/quotes.js]: Please use the shared library sendRequest function

File: package.json (hunk 1 above)
Comment (gibaros) [reply package.json]: Change request performed.

**Pull request merged**